tensorflow/compiler/mlir/BUILD

```diff
@@ -74,6 +74,7 @@ cc_library(
         "//tensorflow/compiler/mlir/tensorflow:tf_legalize_hlo",
         "//tensorflow/compiler/mlir/xla:hlo",
         "//tensorflow/compiler/mlir/xla:hlo_legalize_to_lhlo",
+        "//tensorflow/compiler/mlir/xla:buffer_assignment",
```

Comment:
This needs to be sorted. See the internal CI failure.

Reply:
Done.